stdeb/util.py

```diff
@@ -1205,18 +1208,22 @@ def __init__(self,
             'scripts': scripts
         }
 
+        scripts = ''
```

Comment:
It might be better to keep the type consistent. I'm not sure if mypy is used here, but it would likely complain about this kind of thing.
```suggestion
        scripts = ()
```

Reply:
I gave this a shot but it caused failures which I didn't have energy to investigate further at the moment.